DATA/FX/effects.ini

```diff
@@ -1331,6 +1331,20 @@ nickname = pi_missile02_impact
 effect_type = EFT_EXPLOSION_MISSILE
 vis_effect = pi_missile02_impact
 
+;HYPERION Zoners Heavy Missile
+[Effect]
+nickname = z_missile_heavy01_drive

```

Comment:
Nickname for factions guns starts with the same nick as the faction uses.

In that case its not z_gun it is gd_z_gun...